### PR TITLE
Hacktoberfest/version

### PR DESCRIPTION
**Pull request by voltbit**
Added version information to the report as requested in issue [#39](https://github.com/SUPERAndroidAnalyzer/super/issues/39).

### CODE REVIEW

**Review by Razican — CHANGES_REQUESTED** [review]
Hi! thanks for the changes!
We don't actually need a constant though: SUPER should always be build using cargo, which automatically adds the enviroment variable, and we are using [`clap`](https://crates.io/crates/clap), so we can use the [`crate_version!()`](https://docs.rs/clap/2.14.0/clap/macro.crate_version.html) macro to get it easily, the same way we do it when we create the `clap` help menu in [`src/main.rs#326`](https://github.com/SUPERAndroidAnalyzer/super/blob/master/src/main.rs#L326).

With those changes I can merge the pull request, thanks!

**Comment by voltbit** [comment]
Hello! Thank you for the review, sorry for pulling in master instead of develop. I have committed the change, but I am not sure if it is enough or if I should make another pull request. Sorry for being sloppy, this is my first ever pull request.

**Review by Razican — APPROVED** [review]

**Comment by Razican** [comment]
@VoltBit it looks really good! don't worry about the base branch, I could change it :) let's wait until tests pass and I will merge the pull request. Thanks for your work!

**Pull request merged**
